extras/http2/src/html/example-component/index.html

```diff
@@ -0,0 +1,8 @@
+{% extends 'layouts/application' %}
+{{ macros.css('example-component') }}
+
+<div class="example-component">
+  <p>{{ text }}</p>
+  <p><strong>{{ caveat }}</strong></p>
+  <p><a href="#TODO">{{ link }}</a></p>
+</div>
```

Comment:
I'd suggest moving this file from `example-component/index.html` to `components/example.html`. Wouldn't want devs to have to change or delete that directory.

Reply:
I guess then you'd have to also do that with the associated scss files and js files to match.

Reply:
hmm maybe something more like `components/example-component/index.html` just in the event that there are other partials within `example-component` that bubble up to the index. obviously it won't in my example, but to show a better sense of organization.

Reply:
Or `components/example/index.html`. Just trying to shorten the name along with having a components subdirectory.